egs/wsj/s5/utils/nnet/subset_data_tr_cv.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash +x
```

Comment:
I don't think you mean to check it in with "+x".

Reply:
'+x' does nothing, it can be manually changed to '-x' which enables printing of the expanded bash commands, line by line as they are executed,